routes/user.js

```diff
@@ -150,68 +150,78 @@ router.get('/:id/invitations/in', async (req, res) => {
 });
 
 //POST create a user invitation
-router.post('/:id/invitation', async (req, res) => {
-  const referrer = req.params.id;
-  try {
-    const user = await User.findById(referrer);
+router.post(
+  '/:id/invitation',
+  [check('toEmail', 'Email required').isEmail().trim()],
+  async (req, res) => {
+    const errors = validationResult(req);
 
-    if (!user) {
-      res.status(404).send('User not found');
+    if (!errors.isEmpty()) {
+      return res.status(400).json({ errors: errors.array() });
     }
-    const emails = req.body.emailList;
 
-    //for each email create an invitation instance
-    for (const email of emails) {
-      //determine if referrer already sent invitation to receiver
-      const invitations = await Invitation.find({ toEmail: email });
+    const referrer = req.params.id;
+    const toEmail = req.body.toEmail;
 
+    try {
+      const user = await User.findById(referrer);
+
+      if (!user) {
+        return res.status(404).json({ msg: 'User not found', toEmail });
+      }
+
+      //find all outgoing invitations to the toEmail (array)
+      const invitations = await Invitation.find({ toEmail });
+
+      //see if an invitation was alreasdy sent from user to toEmail
       const invitationAlreadySent = invitations.find(
         (invitation) => invitation.referrer.toString() === referrer,
       );
-
-      //can't send invite if already sent but CAN if the invite was rejected
-      if (invitationAlreadySent && invitation.status !== 'rejected') {
-        console.log('Invitation already sent');
+      //can't send invite if already sent
+      if (invitationAlreadySent) {
+        return res
+          .status(400)
+          .json({ msg: 'Invitation already sent', toEmail });
```

Comment:
Since this is the 'create' function rather than 'send', let's keep the terminology consistent and say 'create' every time you have 'sent' here